firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -164,15 +164,17 @@ void threadBLEScanner(void *param) {
   while(true){
     //filter has to be remade each time, as globalDoorId (from eeprom) is usually read while this thread is already in while loop
     BleScanFilter filter; 
-    char address[18]; 
+    char address[18];
```

Comment:
Thanks for removing the unnecessary spaces!

Reply:
No problem